src/translations/en.ts

```diff
@@ -19,4 +19,13 @@ export default {
     'file-viewer.open-in-fullscreen': 'Open in fullscreen',
     'file-viewer.open-in-new-tab': 'Open in a new tab',
     'file-viewer.more-actions': 'More…',
+    'editor-menu.bold': 'Bold',
+    'editor-menu.italic': 'Italic',
+    'editor-menu.h1': 'Header level 1',
+    'editor-menu.h2': 'Header level 2',
+    'editor-menu.h3': 'Header level 3',
+    'editor-menu.bulleted-list': 'Bulleted list',
+    'editor-menu.numbered-list': 'Numbered list',
+    'editor-menu.blockquote': 'Blockquote',
```

Comment:
Don't we need `'editor-menu.strikethrough': 'Strikethrough',` as well

Reply:
We don't have strike through as part of our menu. We can add it, but it wasn't in the options to be added to the menu when I built the new one.